domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data

```diff
@@ -4,10 +4,10 @@ db.project.insertMany([
     "tenantId": "HR",
     "code": "PR2",
     "name": "Project2",
-    "eatId": "EAT2",
-    "departmentId": "DEP2",
-    "locationId": [
-      "L11","L12"
+    "expenditureId": "910f4d23-fc63-48fb-a8fb-f0ad567b788a",
+    "departmentId": "3d9ef18a-361a-40cf-b142-dd6f998e1ad2",
+    "locationIds": [
+      "bddfe094-9902-4c1e-91dc-1c07c0ce926c","7b8dd41c-c6c1-43f2-9794-ce2e3d648eaa"
```

Comment:
Keep this array empty. We don't have corresponding uuids in the location service data.